src/main/java/DukeException.java

```diff
@@ -0,0 +1,6 @@
+public class DukeException extends RuntimeException {
+
+    public DukeException(String errorMsg){
+        super(errorMsg);
+    }
+}
```

Comment:
Should this be extending `Exception` instead of just `RuntimeException`, so that it can catch more general exception?